tests/test_core.py

```diff
@@ -1116,6 +1116,22 @@ def test_soco_set_player_name(self, moco):
                 ('DesiredConfiguration', '')]
         )
 
+    def test_create_stereo_pair(self, moco):
```

Comment:
I know that the docstring doverage in the tests is spotty, but please add one

Reply:
Calling it 'spotty' is being charitable :)